t/14-isotovideo.t

```diff
@@ -356,6 +399,7 @@ subtest 'publish assets' => sub {
 done_testing();
 
 END {
+    rmtree "./serial0";
```

Comment:
I guess this works, but it looks a bit misleading to me as it is a file and not a directory.
```suggestion
    unlink "./serial0";
```